src/patternfly/demos/Page/examples/Page.md

```diff
@@ -17,6 +17,58 @@ Using the `.pf-m-full-height` modifier class on the page component eliminates th
 {{> page-template page-template--id="page-demo-full-height" page-template--modifier="pf-m-full-height"}}
 ```
 
+### Using flex layout instead of multiple body elements
```

Comment:
If you want to merge this demo (assuming it isn't just for the PR to test), we should update it so it's clear this applies to the sidebar. I'm not sure that we need it as long as we open up the body elements to take any children.